### ISSUE TITLE
allDay error on Android

### ISSUE THREAD

**Issue by amanthegreatone** [wmcmahan/react-native-calendar-events#64]
Hi,

Getting an error on Android when setting allDay: false. The event is getting created but on the emulator the app quits, on the device the app doesn't quit but the error stacktrace is the same.

Android Genymotion emulator (Android 7.0) / Device OnePlus (Android 6.0.1)
RN 0.44.0
Calendar-Events 1.3.6

error stacktrace from android studio:

```
05-18 11:57:13.411 21570-21580/? E/ContentValues: Cannot cast value for allDay to a Integer: false
                                                  java.lang.ClassCastException: java.lang.Boolean cannot be cast to java.lang.Number
                                                      at android.content.ContentValues.getAsInteger(ContentValues.java:288)
                                                      at com.android.providers.calendar.CalendarProvider2.fixAllDayTime(CalendarProvider2.java:1530)
                                                      at com.android.providers.calendar.CalendarProvider2.insertInTransaction(CalendarProvider2.java:2192)
                                                      at com.android.providers.calendar.SQLiteContentProvider.insert(SQLiteContentProvider.java:99)
                                                      at android.content.ContentProvider$Transport.insert(ContentProvider.java:263)
                                                      at android.content.ContentProviderNative.onTransact(ContentProviderNative.java:163)
                                                      at android.os.Binder.execTransact(Binder.java:453)
05-18 11:57:13.412 21570-21580/? E/ContentValues: Cannot cast value for allDay to a Integer: false
                                                  java.lang.ClassCastException: java.lang.Boolean cannot be cast to java.lang.Number
                                                      at android.content.ContentValues.getAsInteger(ContentValues.java:288)
                                                      at com.android.providers.calendar.CalendarProvider2.updateEventRawTimesLocked(CalendarProvider2.java:3024)
                                                      at com.android.providers.calendar.CalendarProvider2.insertInTransaction(CalendarProvider2.java:2202)
                                                      at com.android.providers.calendar.SQLiteContentProvider.insert(SQLiteContentProvider.java:99)
                                                      at android.content.ContentProvider$Transport.insert(ContentProvider.java:263)
                                                      at android.content.ContentProviderNative.onTransact(ContentProviderNative.java:163)
                                                      at android.os.Binder.execTransact(Binder.java:453)
05-18 11:57:13.413 21570-21580/? E/ContentValues: Cannot cast value for allDay to a Integer: false
                                                  java.lang.ClassCastException: java.lang.Boolean cannot be cast to java.lang.Number
                                                      at android.content.ContentValues.getAsInteger(ContentValues.java:288)
                                                      at com.android.providers.calendar.CalendarInstancesHelper.updateInstancesLocked(CalendarInstancesHelper.java:698)
                                                      at com.android.providers.calendar.CalendarProvider2.insertInTransaction(CalendarProvider2.java:2203)
                                                      at com.android.providers.calendar.SQLiteContentProvider.insert(SQLiteContentProvider.java:99)
                                                      at android.content.ContentProvider$Transport.insert(ContentProvider.java:263)
                                                      at android.content.ContentProviderNative.onTransact(ContentProviderNative.java:163)
                                                      at android.os.Binder.execTransact(Binder.java:453)
```

**Comment by wmcmahan** [comment]
Thanks @amanthegreatone, I will try to get this fix in the next release.

**Comment by wmcmahan** [comment]
A fix for this in now in 1.4.0.
